### DISCUSSION TITLE
Mi practica css vacunacion

### DISCUSSION THREAD

**Discussion by Ingrzs** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#308] [Show and tell]
¿Que tal va quedando?, no es tan buena como las que han subido unos aqui pero ahi va, todavia me faltaria lo responsive y los iconos.
![1](https://user-images.githubusercontent.com/94188197/157174518-8f697808-4a0c-4c31-b530-21e6c0f2cc99.png)
![2](https://user-images.githubusercontent.com/94188197/157174600-0943632c-5a1a-4bb4-91b4-71dd07bb116d.png)
![4](https://user-images.githubusercontent.com/94188197/157174653-d934608c-7a1e-4749-81b4-d307069423b6.png)
.

**Comment by GabriielGamez** [comment]
Te va mejor que a mi, honestamente soy un desastre para CSS   :(

**Reply by JuanSalvador73** [reply]
La clave para el dominio de CSS es ir haciéndolo por partes, sobre todo cuando se usa por primera vez. Ya conforme uno va agarrando experiencia, uno sabe donde meter un div con clases dentro de otro casi por "instinto", sin perderse entre tanto código.

**Reply by Ingrzs** [reply]
Pero aun asi me falta a comparación que casi uno los clonaron  igual 😅,Yo vi los vídeos de Soy Dalto en Youtube eso me ayudo mucho, date una vuelta por ahi, entre mas práctica mas le agarras 😅.

**Comment by ArmandoRiveraT** [comment]
Yo no se ni como empezar jajaja. Te está quedando bien tu página 😀👍

**Comment by sergiolopezloya** [comment]
Todo es cuestión de práctica, ya cuando le entiendas mejor podrás usar UIs. son muy útiles, son CSS prefabricados y sólo usas las clases, como Bootstrap o UIkit, ánimo!!
